nixos/modules/installer/tools/nixos-generate-config.pl

```diff
@@ -62,6 +65,12 @@ sub runCommand {
     elsif ($arg eq "--show-hardware-config") {
         $showHardwareConfig = 1;
     }
+    elsif ($arg eq "--dont-configure-user") {
+        $dontConfigureUser = 1;
+    }
+    elsif ($arg eq "--uninteractive") {
```

Comment:
How about `--noninteractive` or `--batch`?

Reply:
thanks noninteractive is better